src/javascript/app/base/client.js

```diff
@@ -47,6 +47,15 @@ const Client = (() => {
             applyToAllElements('.client_logged_out', (el) => {
                 el.setVisibility(1);
             }, '', el_section);
+            applyToAllElements('.client_logged_in', (el) => {
+                el.setVisibility(0);
+            }, '', el_section);
+            applyToAllElements('#client-logged-in', (el) => {
+                el.setVisibility(0);
+            }, '', el_section);
+            applyToAllElements('#topbar-msg', (el) => {
```

Comment:
There's probably only one instance of `#topbar-msg` element so probably don't need to use `applyToAllElements`.